stubs/taint/core_privacy_security/server_side_template_injection_sinks.pysa

```diff
@@ -0,0 +1,29 @@
+# SSTI
+def jinja2.environment.Template.__new__(cls, source: TaintSink[ServerSideTemplateInjection]): ...
+def jinja2.environment.Environment.from_string(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+def jinja2.environment.Environment.compile_expression(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+
+#django.template
+def django.template.engine.Engine.from_string(source: TaintSink[ServerSideTemplateInjection]): ...
+def django.template.base.Template.__init__(source: TaintSink[ServerSideTemplateInjection]): ...
+
+#mako
```

Comment:
Can you move everything from here and below into `stubs/third_party_taint/server_side_template_injection_sinks.pysa`?

Internally we don't use these engines, and having them in this file will result in our testing trying to verify the models against code we don't have, which obviously causes errors. `third_party_taint/server_side_template_injection_sinks.pysa` is where we put models for stuff we don't have stubs or internal code for